### DISCUSSION TITLE
Help me check KEDA Created HPA Status

### DISCUSSION THREAD

**Discussion by yuangeqingian** [kedacore/keda#5987] [Q&A / Need Help]
AKS Version: 1.29.2
KEDA Version:  2.12.1

I created scaleobject for deployment, and it also creates hpa.
scaleobject yaml:
`apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
  finalizers:
  - finalizer.keda.sh
  name: scaledobject-aa-linux-agent
  namespace: ddceado
  resourceVersion: "82626226"
  uid: a36a752c-5acb-4724-a82a-8e0834714c67
spec:
  advanced:
    horizontalPodAutoscalerConfig:
      behavior:
        scaleDown:
          policies:
          - periodSeconds: 30
            type: Percent
            value: 100
          stabilizationWindowSeconds: 300
    restoreToOriginalReplicaCount: true
    scalingModifiers: {}
  cooldownPeriod: 300
  fallback:
    failureThreshold: 3
    replicas: 6
  idleReplicaCount: 0
  maxReplicaCount: 15
  minReplicaCount: 1
  pollingInterval: 5
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: aa-linux-agent
  triggers:
  - authenticationRef:
      name: triggerauth-aa-linux-agent
    metadata:
      demands: ddce-aa-linux-agent,java
      organizationURLFromEnv: AZP_URL
      poolName: BGC-DDCE-Agent
    type: azure-pipelines
  hpaName: keda-hpa-scaledobject-aa-linux-agent
  originalReplicaCount: 1
  scaleTargetGVKR:
    group: apps
    kind: Deployment
    resource: deployments
    version: v1
  scaleTargetKind: apps/v1.Deployment`

The created HPA target showed unknown, don't know if the status is ok.
`NAME                                   REFERENCE                   TARGETS             MINPODS   MAXPODS   REPLICAS   AGE
keda-hpa-scaledobject-aa-linux-agent   Deployment/aa-linux-agent   <unknown>/1 (avg)   1         15        0          44d`

Can you please help check targets showed unknown status. is there any issue on the new installed HPA

**Comment by JorTurFer** [comment]
Hello
That's normal and it's because your workload is scaled to 0. When you have 1 or more instances you will see the value there instead of <unknown>
